src/content/en/fundamentals/media/mobile-web-video-playback.md

```diff
@@ -0,0 +1,611 @@
+      event.stopPropagation();
+      video.currentTime = Math.max(video.currentTime - skipTime, 0);
+    });
+
+As before, rather than adjusting video styling in the `click` event listeners
+of these buttons, we'll use the fired `seeking` and `seeked` video events to
+adjust video brightness. My custom `seeking` CSS class is as simple as `filter:
+brightness(0);`.
+
+    video.addEventListener('seeking', function() {
+      video.classList.add('seeking');
+    });
+
+    video.addEventListener('seeked', function() {
+      video.classList.remove('seeking');
+    });
+
+Here's below what we have created so far. In the next section, we'll implement
+the fullscreen button.
+
+<video controls controlsList="nodownload" muted playsinline>
+  <source src="/web/fundamentals/media/videos/video-play-pause-seek.webm"
+          type="video/webm">
+  <source src="/web/fundamentals/media/videos/video-play-pause-seek.mp4"
+          type="video/mp4">
+</video>
+
+## Fullscreen
+
+Here we are going to take advantage of several Web APIs to create a perfect
+and seamless fullscreen experience. To see it in action, check out the
+[sample]{: .external }.
+
+Obviously, you don't have to use all of them. Just pick the ones that make
+sense to you and combine them to create your custom flow.
+
+<video controls controlsList="nodownload" muted playsinline>
+  <source src="/web/fundamentals/media/videos/perfect-fullscreen.webm"
+          type="video/webm">
+  <source src="/web/fundamentals/media/videos/perfect-fullscreen.mp4"
+          type="video/mp4">
+</video>
+
+### Prevent automatic fullscreen
+
+On iOS, `video` elements automagically enter fullscreen mode when media
+playback begins. As we're trying to tailor and control as much as possible our
+media experience across mobile browsers, I recommend you set the `playsinline`
+attribute of the `video` element to force it to play inline on iPhone and not
+enter fullscreen mode when playback begins.
+
+<pre class="prettyprint lang-html">
+&lt;div id="videoContainer"&gt;
+  &lt;video id="video" src="file.mp4" <strong>playsinline</strong>&gt;&lt;/video&gt;
+  &lt;div id="videoControls"&gt;...&lt;/div&gt;
+&lt;/div&gt;
+</pre>
+
+Caution: Set `playsinline` only if you provide your own media controls or show
+native controls with `<video controls>`.
+
+### Toggle fullscreen on button click
+
+Now that we prevent automatic fullscreen, we need to handle ourselves the
+fullscreen mode for the video with the [Fullscreen API]{: .external}. When user
+clicks the "fullscreen button", let's exit fullscreen mode with
+`document.exitFullscreen()` if fullscreen mode is currently in use by the
+document. Otherwise, request fullscreen on the video container with the method
+`requestFullscreen()` if available or fallback to `webkitEnterFullscreen()` on
+the video element only on iOS.
+
+Note: I'm going to use a [tiny shim] for the Fullscreen API in code snippets below that
```

Comment:
Nobody has yet shipped the unprefixed API, but Edge implements the webkit-prefixed one, so to work roughly everywhere it's still necessary to fall back to both webkit- and moz-prefixed APIs. And there are funny differences in capitalization, many things to get wrong. I recommend https://github.com/sindresorhus/screenfull.js

To roll your own, fall back to `element.webkitRequestFullscreen()`, not `video.webkitEnterFullscreen()`.